frontend/src/components/eventPage/EventPage.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from "react";
+import Header from "../header/Header";
+import DisplayEventInfo from "./DisplayEventInfo";
+import DisplayEventComments from "./DisplayEventComments";
+
+const EventPage = () => {
+  const current_link = window.location.pathname;
+  const current_eventId = current_link.split("/")[2];
+  const [event, setEvent] = useState();
+  const [comments, setComments] = useState();
+  let [changeInEvent, setchangeInEvent] = useState(true);
+
+  useEffect(() => {
+    if (changeInEvent) {
+      fetchEvent();
+      fetchComments();
+    }
+    setchangeInEvent(false);
+  }, [event]);
+
+  const fetchEvent = () => {
+    fetch(`http://localhost:3000${current_link}`)
```

Comment:
Consider storing it as an env variable so that once you deploy your server side, you don't have the risk of missing a url to change

Reply:
To Do:
Store urls in a .env